src/dev-server.js

```diff
@@ -0,0 +1,89 @@
+import 'babel-polyfill';
+import {DockerExecServer,DockerExecClient} from 'docker-exec-websocket-server';
+import Docker from 'dockerode-promise';
+import express from 'express';
+import http from 'http';
+import fs from 'fs';
+import slugid from 'slugid';
+import path from 'path';
+import reactViews from 'express-react-views'
+
+var debug = require('debug')('dev-server:');
+
+const PORT = 34678;
+const DOCKER_SOCKET = '/var/run/docker.sock'
+const PATH = '/'+slugid.v4();
+
+var dockerServer, container, docker,server,dockerClient;
+//check if docker is running i.e socket is available
+var start = async ()=>{
+  debug('Process started');
+  if(!fs.statSync(DOCKER_SOCKET).isSocket()){
+    debug("Docker is not running");
+    throw new Error("Docker is not running");
+  }
+  //Set up the docker server
+  docker = new Docker({socketPath:DOCKER_SOCKET});
+  await docker.pull('ubuntu');
```

Comment:
No need... you have `gdp-test` locally...

What we could do is... we could push `gdb-test` to `ckousik/gdb-test` if you create an account on `hub.docker.com`...
Then if we pull from there using `await docker.pull('ckousik/gdb-test');` we don't have to do `docker build -t gdb-test` anymore... Only when we want to update the image, it should make the project easier to hack and remove two steps from the readme :)

Note: thne you would have to set: `Image: "ckousik/gdb-test"` longer down... in `createContainer`.